### ISSUE TITLE
error: ‘FALSE’ was not declared in this scope

### ISSUE THREAD

**Issue by gabrielvalderramos** [jpbarrette/curlpp#101]
building on PopOS 20.04 LTS

I folowed this steps:
```
cmake -DCURLPP_BUILD_SHARED_LIBS=OFF -DCURL_LIBRARY=../lib -DCURL_INCLUDE_DIR=../include ..
make
```

I got this error:
```
curlpp/internal/OptionSetter.cpp:207:37: error: ‘FALSE’ was not declared in this scope
  207 |  handle->option(CURLOPT_NOPROGRESS, FALSE);
```

Thank you!

**Comment by sgallou** [comment]
Hi,
the problem is known (see [PR#100](https://github.com/jpbarrette/curlpp/pull/100)), and a solution will be provided soon.
As temporary workaround, replacing "FALSE" by "0L" should work.

**Comment by sgallou** [comment]
Fixed in bf49593ad10f1ddd438e58695772ad71d1608e4c
